config/example_config.yml

```diff
@@ -404,14 +402,9 @@ Training:
     # Randomly chosen fraction of data to set aside for validation.
     validation_split: 0.1
 
-    # Required integer.
-    # Number of validations made before finishing training. If 0, run forever.
-    num_validations: 0
-
-    # Required integer.
-    # Number of training steps to run before each evaluation
-    # on the validation set.
-    num_training_steps_per_validation: 1000
+    # Required int.
+    # Number of steps (usually, batches) on which to train.
+    max_steps: 10000
```

Comment:
What do you think of getting the max_steps values from num_epoch * training_steps_per_epoch? I think it's more intuitive and accepted by the DL community to show our metrics vs epochs.

Reply:
Sure, that makes sense to me.

Reply:
Good! The argument should be also renamed to `num_epochs` here, since we are using `config['Training']['num_epochs']` in `run_model.py` line 233.